src/util/irep.cpp

```diff
@@ -160,47 +160,23 @@ const irept &irept::find(const irep_namet &name) const
   return it->second;
 }
 
-irept &irept::add(const irep_namet &name)
+irept &irept::add(const irep_namet &name, bool mark_sharable)
```

Comment:
The flag is kinda critical; any chance of some documentation for it?

Reply:
The documentation is now in the header file.